### PR TITLE
MON-3213: Changing the severity of "missing runbook_url annotation for critical alerts" test case from flaky to failure

### PR DESCRIPTION
**Pull request by vimalk78**
The following critical alerts do not have have runbook_url annotation

| namespace                                                 | alert group name                               | alert                                               |
|-----------------------------------------------------------|------------------------------------------------|-----------------------------------------------------|
| openshift-ovn-kubernetes-master-rules                     | cluster-network-operator-master.rules          | OVNKubernetesNorthboundDatabaseClusterIDError       |
| openshift-ovn-kubernetes-master-rules                     | cluster-network-operator-master.rules          | OVNKubernetesSouthboundDatabaseClusterIDError       |
| openshift-ovn-kubernetes-master-rules                     | cluster-network-operator-master.rules          | OVNKubernetesNorthboundDatabaseLeaderError          |
| openshift-ovn-kubernetes-master-rules                     | cluster-network-operator-master.rules          | OVNKubernetesSouthboundDatabaseLeaderError          |
| openshift-ovn-kubernetes-master-rules                     | cluster-network-operator-master.rules          | OVNKubernetesNorthboundDatabaseMultipleLeadersError |
| openshift-ovn-kubernetes-master-rules                     | cluster-network-operator-master.rules          | OVNKubernetesSouthboundDatabaseMultipleLeadersError |
| openshift-ovn-kubernetes-master-rules                     | cluster-network-operator-master.rules          | OVNKubernetesNorthdInactive                         |
| openshift-kube-scheduler-operator-kube-scheduler-operator | cluster-version                                | KubeSchedulerDown                                   |
| openshift-cluster-storage-operator                        | default-storage-classes.rules                  | MultipleDefaultStorageClasses                       |
| openshift-machine-api-machine-api-operator                | machine-api-operator-metrics-collector-up      | MachineAPIOperatorMetricsCollectionFailing          |
| openshift-machine-config-operator-machine-config-daemon   | mcd-reboot-error                               | MCDRebootError                                      |
| openshift-machine-config-operator-machine-config-daemon   | extremely-high-individual-control-plane-memory | ExtremelyHighIndividualControlPlaneMemory           |
| openshift-ingress-operator-ingress-operator               | openshift-ingress.rules                        | HAProxyDown                                         |

### CODE REVIEW

File: test/extended/prometheus/prometheus.go [hunk 1]
```diff
@@ -80,6 +80,22 @@ var _ = g.Describe("[sig-instrumentation][Late] OpenShift alerting rules [apigro
 		"InstallPlanStepAppliedWithWarnings",
 	)
 
+	criticalAlertsMissingRunbookURLExceptions := sets.NewString(
+		"OVNKubernetesNorthboundDatabaseClusterIDError",
```
[the remark this answers is not in the document]
Comment (vimalk78) [reply test/extended/prometheus/prometheus.go:84]: Added OCPBUGS JIRAs for each exception.

File: test/extended/prometheus/prometheus.go [hunk 2]
```diff
@@ -80,6 +80,39 @@ var _ = g.Describe("[sig-instrumentation][Late] OpenShift alerting rules [apigro
 		"InstallPlanStepAppliedWithWarnings",
 	)
 
+	criticalAlertsMissingRunbookURLExceptions := sets.NewString(
```
Comment (simonpasquier) [inline test/extended/prometheus/prometheus.go:83]: need exceptions for `ClusterOperatorDown` and `ClusterVersionOperatorDown`.

**Comment by simonpasquier** [comment]
@vimalk78 the code doesn't build because you need to remove the import of "github.com/openshift/origin/pkg/test/ginkgo/result" in test/extended/prometheus/prometheus.go.

**Review by simonpasquier — COMMENTED** [review]
/lgtm

**Comment by simonpasquier** [comment]
/hold cancel

Payload jobs are all green.

**Pull request merged**